src/sentry/api/serializers/models/project.py

```diff
@@ -1046,9 +1046,11 @@ def format_options(self, attrs: dict[str, Any]) -> dict[str, Any]:
                 self.get_value_with_default(attrs, "sentry:feedback_user_report_notifications")
             ),
             "sentry:feedback_ai_spam_detection": bool(
-                options.get("sentry:feedback_ai_spam_detection")
+                self.get_value_with_default(attrs, "sentry:feedback_ai_spam_detection")
```

Comment:
Where are the defaults defined?

Reply:
https://github.com/getsentry/sentry/blob/0bb3b6292a8b9e2eb3f30a5ddf3be41e37292d57/src/sentry/projectoptions/defaults.py#L136